bullet_train-outgoing_webhooks/test/setup/active_record.rb

```diff
@@ -1,42 +1,12 @@
-require "active_record"
-
-ActiveRecord::Base.establish_connection(adapter: "sqlite3", database: ":memory:")
-
 # Set so Active Record casts created_at/updated_at to ActiveSupport::TimeWithZone.
-# (Mathces Rails apps' default, but its assigned via Active Record's railtie.)
+# (Matches Rails apps' default, but its assigned via Active Record's railtie.)
 Time.zone = "UTC"
 ActiveRecord::Base.time_zone_aware_attributes = true
 
-ActiveRecord::Schema.define do
-  create_table :teams, force: true do |t|
-    t.string :name, null: false
-    t.timestamps null: false
-  end
-
-  create_table :webhooks_outgoing_endpoints, force: true do |t|
-    t.references :team
-    t.string :url, null: false
-    t.string :name, null: false
-    t.integer :event_type_ids, array: true, default: []
-    t.timestamps null: false
-  end
-
-  create_table :webhooks_outgoing_deliveries, force: true do |t|
-    t.references :team
-    t.references :endpoint
-  end
-end
-
-class Team < ActiveRecord::Base
-  has_many :webhooks_outgoing_endpoints, class_name: "Webhooks::Outgoing::Endpoint"
-end
-
-class Webhooks::Outgoing::Delivery; end
-
-5.times do |n|
-  Team.create! name: "Generic name #{n}"
-end
-
-5.times do |n|
-  Webhooks::Outgoing::Endpoint.create! team_id: Team.first.id, name: "Generic name #{n}", url: "http://example.com/webhook-#{n}"
+# We declare a Team model here so that we can have it
+# include Webhooks::Outgoing::TeamSupport
+# which we don't ship enabled by default. Maybe we should?
```

Comment:
Interesting, I would imagine we should ship it by default, especially since all of the Developers > API/Webhooks logic is available on a new Bullet Train app out of the box.

Reply:
I just double checked the starter repo and there we _do_ include `Webhooks::Outgoing::TeamSupport` by default. It's just the copy in the `core` repo that doesn't include it. I'll try adding it there and see what happens.

Reply:
Nope, no luck. Turns out that the team model is in the `bullet_train` gem, and that gem doesn't depend on the `outgoing_webhooks` gem, so that module isn't always available. (Relatedly, we should probably take a look at the dependency graph that we've constructed for all of these gems and make sure that what we currently have is actually what we want. I'm not sure we have a fully accurate story about which gems are really a "core" part of the BT stack vs which ones just happen to live in the `core` repo. But that's for another issue/PR.)